### DISCUSSION TITLE
Stuck with electron detector

### DISCUSSION THREAD

**Discussion by testuser83** [ozel/DIY_particle_detector#41] [Q&A]
Hi,

I really like what you have done and have tried to make an electron detector. I am now trying to figure out how to get it working. I have a piece of uranium glazed tile from my 1930's house that gives a strong signal on my Geiger counter and I should be able to detect Uranium decay as it is a  beta emitter. The web oscilloscope does not seem to work for me. I rarely see a signal, bu most of the sime it says waiting for signal. So I switched to WaveWatch and I get this signal:

![wavewatch signal](https://user-images.githubusercontent.com/106806530/171824944-c4d8a70a-7a1a-4f8c-8123-94f35553bfa8.jpg)

Assuming this is good I wanted to export, but I don't know how WaveWatch can save a file and the documentation website does not exist anymore..

So I used Theremino. As I understand it, Theremino is already a full spectrometer. I measured two 3 minute graphs, first this one with the Uranium tile:

![Theremino with radioactive source](https://user-images.githubusercontent.com/106806530/171826058-0950b0d5-9e0a-45a3-81b9-1d1b7c0b259e.jpg)

Then I recorded a graph without any source in the tin box:

![Theremino without radioactive source](https://user-images.githubusercontent.com/106806530/171826612-17bcff1f-a97a-4616-82d2-b6eec066ddec.jpg)

And here is the overlay of both:

![Theremino overlay](https://user-images.githubusercontent.com/106806530/171826869-c8db4a97-7d81-491c-ab15-8f181f49e90a.jpg)

I think there is definately something measured and the tile adds quite a lot on top of the no tile signal.

And here is where I am stuck. Is this a valid recording and how to proceed? 

Regards,
Eric.

**Comment by ozel** [comment]
Hi Eric,
Your first screenshot of WaveWatch is the most important one. It looks like there is way too much noise and no real signal. 
You should not use any spectral software like Theremino before you see clear individual pulses as shown here: 
https://github.com/ozel/DIY_particle_detector/wiki/Software#processing-based-audio-oscilloscope

Making sure that your detector works correctly is easier on a real oscilloscope, in case you can get access to one somewhere (visiting local maker spaces or fab labs on their weekly open day usually does the trick).
https://github.com/ozel/DIY_particle_detector/wiki/Oscilloscope-Measurements

The problem with using soundcards as oscilloscopes is that you first need to find a good volume/gain setting in the system settings of your operating system. Usually, it should be set rather high, but not too high. Which sound card/microphone input are you using?

At first, we have to make sure that your detector is not picking up noise because it is maybe not properly mounted in a metal case or not shielded well from ambient light. Can you please post here at least two pictures of your detector that show it in an opened and closed metal box? (just drag & drop pictures into this text field)

Also please go through this list of troubleshooting tips from top to bottom and let me know at which point you get stuck:
https://github.com/ozel/DIY_particle_detector/wiki/Troubleshooting

Using a piece of Uranium glazed tile with confirmed radioactivity is already a very good starting point for testing your electron detector.

**Comment by testuser83** [comment]
Hi Ozel,

Thank you for the quick response. I could not find access to an oscilloscope, but what you say sounds very sensible. So I focused on the sound part.

I am using a C Media soundcard based on CM108: https://www.amazon.nl/gp/product/B099DZYV5Z/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&psc=1

Here is my PCB:
![IMG_20220604_131648](https://user-images.githubusercontent.com/106806530/171998031-e442ef87-2e7b-49c9-a5ce-a567c21dd3a8.jpg)

Here it is inside the tin can:
![IMG_20220604_134211](https://user-images.githubusercontent.com/106806530/171998052-4ce26997-1209-4acd-955f-b77b64d4a3af.jpg)

And here with the lid closed:
![IMG_20220604_134248](https://user-images.githubusercontent.com/106806530/171998580-c82ca1bf-e009-4661-a69a-8b5fa701d7ed.jpg)

I enabled listening on the sound card so you can hear my signal:
[recording.zip](https://github.com/ozel/DIY_particle_detector/files/8837657/recording.zip)

What you hear is me turning the device on, waiting for a bit, touching the lid a few times and then opening the lid. Notice that the signal is gone after opening the lid. The hum worries me, but I cannot find a ground fault or something like it. 

I also tested it on my PC soundcard and the result is the same.

I followed the troubleshooting steps and could not find any faults.

**Comment by ozel** [comment]
Hi Eric,

thanks for the pictures, the soldering seems fine as far as it can be seen (somewhat blurry). Could you post another picture of the backside of the PCB, showing the diodes?

Since you chose the clear BPW34 diodes, you need to take extra care that no light reaches inside the metal box (the one you have is a good choice to start with but if you are in a very bright room, light
 could reach inside for example where the cable passes by the lid).
If light leaks inside, the diodes could easily pick up 50/60 Hz or multiples of that from artificial, mains-connected light sources (fluorescence lights, LED etc.).
Put some dark cloth, like several pullovers, on top of the box and check if the humming noise is reduced.

You could also have a faulty/noisy power supply with your computer. Try to connect the USB sound card to a laptop and run it just from the laptop's internal battery - also disconnect any other cables from the laptop (they could act as antennas, picking up electromagnetic noise). 
Sometimes it can also help to change the room or try it even in another building on a different computer.
A high-power machine close by like an electric motor or fridge may create electromagnetic interference (EMI) radiation that the sensitive circuit is picking up.
Btw, there is a reason why I recommend mounting a BNC socket in the wall of the tin can together BNC cables: They are shielded which protects against EMI. The mini-jack cable you use may most likely be not shielded.
More on cable recommendations here https://github.com/ozel/DIY_particle_detector/wiki/Cables

Have you checked that you connect to the correct pins of the sound card?
You need a 3-pin mini-jack cable (not 4-pin like for laptops or mobile phones) for the CM108.
Have you checked with a real analog microphone that you can record sound without this humming on your CM108?
I checked your recording in Audacity (free software) and in the "Analyze/Plot Spectrum" function one can see that the humming noise consists of many different frequencies that are very strong, even several multiples of 1000 Hz:
![Screenshot from 2022-06-14 23-35-21](https://user-images.githubusercontent.com/653212/173693398-72addee5-caf7-4c1d-a8fc-58fb21d5041a.png)


In principle, the CM108 is a good sound card choice. If we manage to confirm with more clear pictures that your wiring to the sound card and all components are correctly soldered, we should make it work.
At which microphone/input volume settings did you set the CM108? 
Try reducing it by 50%, it might be set too high in your case.
And which operating system do you use?
Windows may add software amplification and unhelpful processing of sound, intended for voice/video calling.
Have you checked the system settings for something like that?

**Reply by testuser83** [reply]
Hi Ozel, 

Here it is 
![1655291946370](https://user-images.githubusercontent.com/106806530/173816044-128ed4f4-7646-43e4-a131-0aa292c7a3e5.jpg)
my PCB's back with the diodes.

I did some further testin with a thick cloth. I also put a thick metal bin upside down on top of it. I can see or hear no difference.

Instead of my pc, I used a laptop on battery power. I get exactly the same results. I listened to the sound too and it is also the same as the mp3 I sent earlier. Different locations also do not do anything different.

I tested with a stereo microphone and I can hear myself talking. The sound is crips and the hum is not present. I also disconnected the electron detector and listened again. There is very little noise.

There is also very little noise when I open the lid of the tin can. Once closed, the noise comes back. This also happens when I cover the opened tin can with my hands.

I also checked that the audio connector does not shirt circuit. I checked that the signal and ground are correctly connected.

I tried volume settings from no AGC and almost 0 to AGC and 100 %. There is always the humming sound. I think you are correct that it is larger in amplitude than any signal that might occur. 

I am using WIndows 11 on my pc and Windows 10 on the laptop. I made sure the extra processing (which is indeed an option in my sound settings, is turned off on my pc. The laptop does not have any extra processing.

I remeasured al resistors, the are good. I also measured the capacitors. I found one that I suspected faulty and replaced it, but no luck. All diodes measure  ok. I have no way of testing the ic. I also re-soldered the battery and sound connection.

**Reply by ozel** [reply]
did you progress with this? I somehow forgot to follow up.
That the noise changes and gets quiet when you open the box is a very good sign, it means the amplifier gets oversaturated by all the light!

I would still recommend to improve the cabling between soundcard and the box. Make a hole and mount a proper connector in the box (BNC or 3.5mm socket with good ground connection to the box _plus_ a separate wire going to the board ground/minus)

You have not calrified if you use a _3-pin_ or _4-pin_ 3.5 mm mini-jack audio cable. It should only be a _3-pin_ cable with the correct wiring of two pins - leaving one pin unconnected. See the project wiki page about cables.

**Reply by ozel** [reply]
your very first pulse picture at the top of this page looks actually like one real pulse is preset in the center, it is just overlaid with that humming noise (sine waves) that you need to get rid of. Usually with better ground connections and the correct 3-pin mini jack wiring at the sound card.

**Comment by testuser83** [comment]
That is good news. I had already given up, but will give it a try after the holidays. I think my audio connection is ok, but the wires are unshielded.

**Reply by ozel** [reply]
could you please post a picture of the mini-jack connector's solder joints that plugs into the soundcard? 
I mean the soldered wire connections that are hidden under the black plastic part of the mini-jack connector. 
In case it is a pre-assembled/injection-molded connector, please let me know which wire goes where on the 3-pin sections of the red connector on the soundcard
